### ISSUE TITLE
Overall Issue with the working of the App

### ISSUE THREAD

**Issue by Skandesh2008** [25huizengek1/ViTune#534]
### Steps to reproduce the bug

None of the Songs I haven't made offline are playing , Unable to search new songs , none of the images are loading , lyrics are no longer available 

### Expected behavior

It's not working . I don't know why 

### Actual behavior

Sometimes app won't even open 

### ViTune version

V.1.1.1

### What kind of build are you using?

Release (GitHub / F-Droid)

### Android version

Android 14 

### Device info

I am using Readmi A3X

### Upstream reproducibility

Yes I noticed it in other app as well 

### Checklist

- [X] I am able to reach YouTube Music through the **official** app/website and can confirm that I live in a [supported country](https://support.google.com/youtubemusic/answer/6313540?hl=en) (and this country is not temporarily unavailable)
- [X] I am not using a proxy, VPN, device policy or anything else that restricts access to YouTube Music
- [X] This issue has not been submitted already (this is not a duplicate of another issue)

**Comment by 25huizengek1** [comment]
@Skandesh2008 Please provide some logs/a screen recording, I can't know what's going on on your device without it.

**Comment by guest3301** [comment]
I was also having the same issue until now.Its fixed now...
still I'm attaching the logs, see if it helps in debugging;
[debug_logs_redmi.txt](https://github.com/user-attachments/files/17565361/debug_logs_redmi.txt) - Redmi 8A, running on android 10

I just reinstalled the app and it was fixed. thought dns settings might be causing something, changed it to automatic from off, and vice versa, both the times it worked. also I checked the DB to ensure thumbail urls are really being fetched, urls were already there in the tables (when thumbnails were falling back to default one);
when I installed this app on a new device, the issue was gone,
OnePlus 11R 5G running on android 14, debug logs, see if it helps: 
[debug_log_oppo.txt](https://github.com/user-attachments/files/17565492/debug_log_oppo.txt)

@Skandesh2008 Have you tried reinstalling the app? Did it work?

**Comment by Skandesh2008** [comment]
Hi ! Thanks for reaching out .. I am glad my problem is being fixed , First , I have provided a Google photos link to a screen recording which describes the issue I was facing , like songs not playing , search not working ( and has logs)  , discover was not loading , there were no thumbnails , no lyrics ,only offline songs could be played . 

I reinstalled the app , and turned off a few features like quick pics , android auto and persistance que , this helped to bring back the logs , however still the thumbnails are not loading , they are not even being fetched cus the database doesn't show any increment in the data size .. hope you guys fix it as soon as possible .
https://photos.app.goo.gl/y6roJ644YBx7S5FP7

**Comment by Skandesh2008** [comment]
Yo ! Just wanted ro give an update .. I guess I have found the reason  for facing the issue ...
Let me break it down . 
So I noticed that no songs would play until the quicks pics loaded ( it was taking upwards of 10 minutes before ) where  I would lose my patience and close the app . But I let it through one time and after 15 minutes , all songs started playing normally . 
So after the quick pics are loaded , the app works normal ( except there are no thumbnails ) , the search works , discover works and almost everything is normal . 
Next I set the quick picks from trending to last played .. and now it's better evern tough it takes like 2 minutes to load .. but after that it's normal I don't face any lag while playing songs 
And also please note that I have no internet issues , I am running android 14 on a Readmi A3 phone with GO UI ...
Please find a solution and narrow down the wait time for me + bring back the good old thumbnails 
Regards

**Comment by sidsharths1245** [comment]
Was happening with me to, i just uninstalled the app completely and reinstalled it again and it was fixed. Don't forget to export your data beforehand though

**Comment by Skandesh2008** [comment]
Yeah I did that that only temporarily fixed it though + I have a lot of songs , is that of any problem ?

**Comment by 25huizengek1** [comment]
I'll need some logs. I can't work with any of this information. It even seems impossible to me, because when you don't open the quick picks, the app continues to function and none of the page will be loaded.

**Comment by Skandesh2008** [comment]
Sorry I had been gone for a while .. I have noticed something really strange.. see if I use a vpn , all the problems are gone .. I am flabbergasted bruh .. in my normal wifi I don't got songs being played , thumbnails loading and stuff , but under a vpn , it works like butter .. but I wonder what the problem is cus I live in India where yt music is supported .
Here is a link to the screen recording 
https://photos.app.goo.gl/aHqtdVeFJ7wkXdtd8

**Comment by 25huizengek1** [comment]
ViTune is going into maintenance mode. Only security (or related) issues will be accepted. All current open issues are being closed.
